spec/routines/newflow/ensure_unverified_user_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+module Newflow
+  describe Newflow::EnsureUnverifiedUser, type: :routine do
+    let(:subject) do
+      FactoryBot.create(:user, state: 'needs_profile')
+    end
+
+    it 'changes the user state to unverified' do
+      described_class.call(subject)
+      expect(subject.state).to eq(User::UNVERIFIED)
+    end
+
+    it 'outputs the user' do
+      expect(described_class.call(subject).outputs.user).to be_a(User)
+    end
+
+    it 'creates a security log' do
+      expect {
+        described_class.call(subject)
+      }.to change {
```

Comment:
consider using begin/end for multi-line blocks (lines in an editor) 

> Prefer {...} over do...end for single-line blocks. Avoid using {...} for multi-line blocks. Always use do...end for “control flow” and “method definitions” (e.g. in Rakefiles and certain DSLs). Avoid do...end when chaining.

from https://shopify.github.io/ruby-style-guide/ and most other ruby style guides.